index.html

```diff
@@ -6,66 +6,65 @@
     <title>Weather</title>
     <link rel="stylesheet" href="style.css">
     <script src="https://code.jquery.com/jquery-3.7.1.min.js" integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/luxon@3.5.0/build/global/luxon.min.js"></script>
     <script src="script.js"></script>
 
 </head>
 <body>
 
 <div class="card">
 
-    <!--Search bar - does not function - yet...-->
-    <!--Need to learn city->lat/long co-ords to use search-->
-    <div class="search">
-        <input type="text" placeholder="enter city name" spellcheck="false">
-        <button><img src="images/search-icon.png"></button>
+    <!--Search bar-->
+    <div class="searchBar">
+        <input type="text"  placeholder="enter city name" id="searchInput" spellcheck="false">
+        <button><img src="images/search-icon.png" id="searchButton"></button>
     </div>
```

Comment:
If you wrap this in a `<form>` element, you can get hold of it with jQuery and access the submit event so something like:

```
    <form class="searchBar">
        <input type="text"  placeholder="enter city name" id="searchInput" spellcheck="false">
        <button type="submit"  id="searchButton"><img src="images/search-icon.png"></button>
    </form>
```
```
const form = $('#searchBar');
form.on("submit", function(event) = {
   event.preventDefault(); // this stops the page refreshing as a form's default action will send the data
   const searchTerm = $('#searchInput').val(); // gets you the search input string
   /// then here you could call off to a function to go and get the lat/lon, something like:
   getLocation(searchTerm);
})
```